src/commands/ImportCommand.js

```diff
@@ -0,0 +1,83 @@
+import fs from "fs";
+import path from "path";
+import child from "child_process";
+import Command from "../Command";
+import progressBar from "../progressBar";
+import PromptUtilities from "../PromptUtilities";
+
+export default class ImportCommand extends Command {
+  initialize(callback) {
+    const inputPath = this.input[0];
+
+    if (!inputPath) {
+      callback(new Error("Missing argument: Path to external repository"));
+    }
+
+    const externalRepoPath = path.resolve(inputPath);
+
+    try {
+      const stats = fs.statSync(externalRepoPath);
+      if (!stats.isDirectory()) {
+        throw new Error(`Input path "${inputPath}" is not a directory`);
+      }
+      const packageJson = path.join(externalRepoPath, "package.json");
+      const packageName = require(packageJson).name;
+      if (!packageName) {
+        throw new Error(`No package name specified in "${packageJson}"`);
+      }
+      this.targetDir = "packages/"+packageName;
```

Comment:
Sure, though I've tended not to when concatenating a single string constant with a single variable, since a template string is actually longer in that case.  How about I add some space around the operator?  That's the same length, but involves less punctuation.  Seems to [fit](https://github.com/lerna/lerna/blob/master/bin/lerna.js#L43) [with](https://github.com/lerna/lerna/blob/master/src/Command.js#L23) [context](https://github.com/lerna/lerna/blob/master/src/GitUtilities.js#L12), too.